### DISCUSSION TITLE
Committer identity Unknown

### DISCUSSION THREAD

**Discussion by mamgbigeofrey** [community/community#50235] [Actions] [answered]
### Select Topic Area

Question

### Body

Why do I get the error , committed identity  unknown  when trying  to pull or push to my repository. 
Previously  everything  was fine untill when I moved a repository  to my organization  account  of which I am the owner,  but now every action to commit or pull gives me the error specified,  any help

**Comment by Anmol-Baranwal** [comment, marked as answer]
The "committed identity unknown" error usually occurs when Git is unable to associate your commits with a valid identity (i.e., your username and email address). This can happen for a few different reasons, such as:

1. Your Git configuration is not set up correctly: Check that your Git configuration is set up with your correct username and email address. You can use the following commands to check and set your configuration:

```python
git config --global user.name "Your Name"
git config --global user.email "your.email@example.com"
```

Make sure that the email address you use matches the email address associated with your GitHub account

2. Your SSH key is not associated with your account: If you're using SSH to authenticate with GitHub, make sure that your SSH key is associated with your account. You can check this by going to your account settings on the GitHub and verifying that your SSH key is listed.

3. Your repository permissions are not set up correctly: If you recently moved your repository to a new organization account, it's possible that your permissions are not set up correctly. Make sure that you have the appropriate permissions to push and pull from the repository.

Additionally, you may want to reach out to the [GitHub support](https://support.github.com/contact?tags=docs-generic)

**Reply by Anmol-Baranwal** [reply]
@mamgbigeofrey 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by mamgbigeofrey** [comment]
Finally following your solution has cleared my error, and the hustle of having to resolve errors before committing to github is gone, thanks a lot

**Comment by elromklf** [comment]
i am so happy it works for me too
